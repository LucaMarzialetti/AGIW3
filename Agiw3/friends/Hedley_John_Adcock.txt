--url_fb https://www.facebook.com/hedley.adcock
--friends
VM10123:3 Elisabeth Conner
VM10123:3 Tamara Sandbach
VM10123:3 Victoria Khandker
VM10123:3 Tessa Rhodes
VM10123:3 Matt Price
VM10123:3 Jonathon Rosser
VM10123:3 Sebastian Jones
VM10123:3 Adam Price
VM10123:3 Chris Baker
VM10123:3 Samantha Louise
VM10123:3 Paul Coates
VM10123:3 Sam Grant
VM10123:3 Rich Smith-Manifold
VM10123:3 Maria Oettl
VM10123:3 Janet Price
VM10123:3 Jon Huskisson
VM10123:3 Andreas Neumann
VM10123:3 Peter Raybould
VM10123:3 Charlotte DeRenzy Jones
VM10123:3 Mark Adcock
VM10123:3 Alex Carlton
VM10123:3 Janet Bibb
VM10123:3 Alison Broadley
VM10123:3 Lucy O'Dwyer
VM10123:3 Jessica Louise Beddows
VM10123:3 Lisa Moore
VM10123:3 Trish Manning
VM10123:3 Fiona Pagan
VM10123:3 Matt Beckerleg
VM10123:3 Nick Adkins
VM10123:3 Adam James
VM10123:3 Carolin Rodler
VM10123:3 Frances Adcock
VM10123:3 Jen Al-Janabi
VM10123:3 Lauren Powell
VM10123:3 Dario Di Pierri
VM10123:3 Rachel Baker
VM10123:3 Philip Denley
VM10123:3 Mark Steven Vipond
VM10123:3 Mark Terrar
VM10123:3 Graeme Black
VM10123:3 Gemma Allen
VM10123:3 Georgina Davies
VM10123:3 Karen Install
VM10123:3 Wendy Leach
VM10123:3 Rob Coulson
VM10123:3 Robert Leckie
VM10123:3 Tim Edds
VM10123:3 Haidee Jayne
VM10123:3 Michelle Bajcar
VM10123:3 Becka Kelly
VM10123:3 Matthew Green
VM10123:3 Mark Chapman
VM10123:3 Stefan Vieweg
VM10123:3 Jackie Royal
VM10123:3 Mark Shenton
VM10123:3 Emma-Louise Green
VM10123:3 Alexia Haines
VM10123:3 Amanda Godfrey-Mander
VM10123:3 Peter Breakwell
VM10123:3 Steven Robson
VM10123:3 Aaron Marc Georgiou
VM10123:3 Fiona Holloway
VM10123:3 Søren Kjellberg Ishøy
VM10123:3 Richard Broomhall
VM10123:3 Richard Roberts
VM10123:3 Clair Yates
VM10123:3 Paul Davis
VM10123:3 Alexis Aaron Georgiou
VM10123:3 Monica Dabasia
VM10123:3 Dean M Jackson
VM10123:3 Julie Nonna B Ball
VM10123:3 Thy Do
VM10123:3 Lion O Thundera
VM10123:3 Phill Lewis
VM10123:3 Mike Pate
VM10123:3 Claire Smith
VM10123:3 Iain Harris
VM10123:3 Harriet Drinkwater
VM10123:3 Tessa Armstrong
VM10123:3 Bryan Manley-Green
VM10123:3 Lynne Sarjant
VM10123:3 Katharine Anne Bryant
VM10123:3 Robert Richards
VM10123:3 Gemma McGeown
VM10123:3 Steven J Knott
VM10123:3 Paula Louise Young
VM10123:3 Kelly Plant
VM10123:3 Samantha Forrest
VM10123:3 Simon Thomas
VM10123:3 Jessica Rachel
VM10123:3 Rachel Asson
VM10123:3 Darren Yimyam Ball
VM10123:3 Rhiannon Dando
VM10123:3 Alfonsus Masail
VM10123:3 Rachael Sogeler
VM10123:3 John Clarke
VM10123:3 Bal Sangha
VM10123:3 Lee Brown Browny
VM10123:3 Suky Dale
VM10123:3 Damian Nikolic
VM10123:3 Kirsty Frazer
VM10123:3 Tim Edwards
VM10123:3 Jenna Bruce
VM10123:3 Connor Rowe
VM10123:3 Hayley Holdam
VM10123:3 Stella Wheatley
VM10123:3 Ann Adcock
VM10123:3 Hayley Newcombe
VM10123:3 Xander Francies
VM10123:3 Charlotte Moore
VM10123:3 Emma Lucas
VM10123:3 Matthew Smith
VM10123:3 Pav Jr Sandhu
VM10123:3 Jacob Sandbach
VM10123:3 Charis Grimason
VM10123:3 Michelle Louise Whittaker
VM10123:3 Dan Cookson
VM10123:3 Rebecca Bufton
VM10123:3 Rupa Ladwa
VM10123:3 Jochen Troeger
VM10123:3 RS Salvador
VM10123:3 York Massenbach
VM10123:3 Andy Poulton
VM10123:3 Alex Provan
VM10123:3 Charlie Roberton
VM10123:3 Kathryn Morgans
VM10123:3 King Leopold
VM10123:3 Anna Mitchell-Martin
VM10123:3 Princess K Kaur
VM10123:3 Adam Azarzar
VM10123:3 Kultar Deol
VM10123:3 Marc Gould
VM10123:3 Graeme Pate
VM10123:3 Stewart Denley
VM10123:3 Stuart Marsh
VM10123:3 Victoria Hutchings
VM10123:3 Hayel St Croix
VM10123:3 Rosa Di Pierri
VM10123:3 Gareth Hughes
VM10123:3 Nicholas Ford
VM10123:3 Thom Timmer
VM10123:3 Liz Perkins
VM10123:3 Carlene Lowe
VM10123:3 James Bissett
VM10123:3 Renata Carrão
VM10123:3 Laura Brosnan
VM10123:3 Carl Thomas
VM10123:3 Pip Sansom
VM10123:3 Mark Noble
VM10123:3 Andrew Ward
VM10123:3 Caroline Hofer
VM10123:3 Minakshi W
VM10123:3 Mark Bissett
VM10123:3 Olivia Di Punzio
VM10123:3 Mark St John Paget
VM10123:3 Don Alberto
VM10123:3 Boris Pineles
VM10123:3 Hilda Kinnis Robson
VM10123:3 Beatrix Tröger
VM10123:3 Adrian Leonard
VM10123:3 Neil Miles
VM10123:3 Anna Pendlebury
VM10123:3 Adam Nadge Tomson
VM10123:3 Lisa Burford
VM10123:3 Steve Hopkins
VM10123:3 Matthew Smith
VM10123:3 Daniel Braun
VM10123:3 Alba Calmarza
VM10123:3 Al Hodgetts
VM10123:3 Kimberley Thomas
VM10123:3 Emily Earley
VM10123:3 Stephanie Kirby
VM10123:3 Michael Shenton
VM10123:3 BR Hughes
VM10123:3 Rebecca Swales
VM10123:3 Sarah Field
VM10123:3 Jody Kite
VM10123:3 Anna Olesińska
VM10123:3 Kathleen Mary Simpson
VM10123:3 Louise Deans
VM10123:3 Hayley Barnes
VM10123:3 Christoph Muschallik
VM10123:3 Heidi Tierney
VM10123:3 Jamie Sherlock
VM10123:3 Mahmoon Rashid
VM10123:3 Korrine Pallas
VM10123:3 Rory Barrett
VM10123:3 Emma Rabicano
VM10123:3 Fernanda Lima
VM10123:3 Katie Mitchell
VM10123:3 Karly Wright
VM10123:3 Tina Awan
VM10123:3 Sam Hancox-Heal
VM10123:3 Ruth Cutting
VM10123:3 Rachel Thomas
VM10123:3 Elizabeth Butler
VM10123:3 Oliver Cole
VM10123:3 Linzi Kavanagh-Bromley
VM10123:3 Hazel Prichard
VM10123:3 Michelle McCarthy
VM10123:3 Keith Enright
VM10123:3 Julie Back
VM10123:3 Sheryl Moore
VM10123:3 Katie Connor
VM10123:3 Wayne Taylor
VM10123:3 Simon Richardson
VM10123:3 Rebecca Davies
VM10123:3 Claire Chapman
VM10123:3 Victoria Graham
VM10123:3 Martin Hemming
VM10123:3 Jerome Mowbray Holt
VM10123:3 Becky Gardener
VM10123:3 Phil Chapman
VM10123:3 William Schofield
VM10123:3 Richard Embury
VM10123:3 Thomas M Cole
VM10123:3 Hayley Holdam
VM10123:3 Mary Hodgetts
VM10123:3 Gemma Clayton
VM10123:3 Daniel Quinn
VM10123:3 Katie Taylor
VM10123:3 Paul Errington
VM10123:3 Gareth David Williams
VM10123:3 Danielle Boyce
VM10123:3 Matthew DeRenzy Jones
VM10123:3 Paul Dickinson
VM10123:3 Mark O'Donohue
VM10123:3 Steve Carpenter
VM10123:3 Charlotte Fay Peterson
VM10123:3 Sam Storey
VM10123:3 Charlotte Gibson
VM10123:3 Rupa Suresh
VM10123:3 Amanda Dunn
VM10123:3 Jonathon Richards
VM10123:3 Caroline Brown
VM10123:3 Petrina Liyanage
VM10123:3 Richard Brice
VM10123:3 Keil Day
VM10123:3 Ross Miller
VM10123:3 Kristian Waterhouse
VM10123:3 Kirsty McIntyre
VM10123:3 Justine Hurley
VM10123:3 Andrew Deakin
VM10123:3 Wes Powell
VM10123:3 Jane Elizabeth Gaunt
VM10123:3 Emma Westbury
VM10123:3 Jenni Carlin
VM10123:3 Luke Joseph Thomas Plimmer
VM10123:3 Melissa Bennett
VM10123:3 Hayley Brampton
VM10123:3 Nicola Murfin
VM10123:3 Luke Denley
VM10123:3 Stephen Sweeney
VM10123:3 Rhian Blaszkowicz
VM10123:3 Thomas Jolly
VM10123:3 Maren Clare
VM10123:3 Toffi Berger
VM10123:3 Emma Gibson
VM10123:3 Andy Hollis
VM10123:3 Sarah Belshah
VM10123:3 Philip Lee Thompson
VM10123:3 Polly Therese Iddon
VM10123:3 Victoria Newey
VM10123:3 Richard Herdman
VM10123:3 Nicola Lomas
VM10123:3 Clare Haines
VM10123:3 Ryan Hawley
VM10123:3 Jonathan Hope
VM10123:3 Sarah Edwards
VM10123:3 Lister R Metcalfe
VM10123:3 Catherine Puls
VM10123:3 Zoe Bennett-Brock
VM10123:3 Ruth Elen James
VM10123:3 Kathryn Golightly
VM10123:3 Gemma Smith Nee Bostock
VM10123:3 Hayley Aldrich
VM10123:3 Rachel Louise Booth
VM10123:3 Ian Crooks
VM10123:3 Mike Brown
VM10123:3 Stephen Trenery
VM10123:3 Megan Gracie
VM10123:3 Charlie Winship
VM10123:3 James Binding
VM10123:3 Oliver Holloway
VM10123:3 Shelley Fryer
VM10123:3 James Baker
VM10123:3 Ryan McKnight
VM10123:3 Fay Linton-James
VM10123:3 Helen Day
VM10123:3 Toni Shukla
VM10123:3 Chris Barley
VM10123:3 Tracie Robbo
VM10123:3 Neil Denley
VM10123:3 Deia Nguyen
VM10123:3 Dave Archer
VM10123:3 Stewart Denley
VM10123:3 Hayley Frost
VM10123:3 Michael Stevenson
VM10123:3 Michelle Turner
VM10123:3 Helen Whitehouse
VM10123:3 Valentine Taylor Rabin
VM10123:3 Georgina Holloway
VM10123:3 James Blower
VM10123:3 Anthony Macpherson
VM10123:3 Russell Brown
VM10123:3 John Raeburn
VM10123:3 Jennie Bradley
VM10123:3 Antonia Edie
VM10123:3 Mark White
VM10123:3 Emma Hawkes
VM10123:3 Gemma Green
VM10123:3 Gareth Powell
VM10123:3 Andrew Yates
VM10123:3 Tristan Jones
VM10123:3 Katie Niblett
VM10123:3 Nico Bellucci
VM10123:3 Michael Adams
VM10123:3 Robert Lamb
VM10123:3 Beth Adams
VM10123:3 Jay Mukhey
VM10123:3 Andrew Roper
VM10123:3 Lorraine Traill
VM10123:3 Tim Hapgood
VM10123:3 Rhiannon Hillage
