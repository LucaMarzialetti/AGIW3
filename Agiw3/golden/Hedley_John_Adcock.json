{
    "PATTERN": {
        "tel": ["08454 70 80 81",
            "+44 (0)20 7320 5757"],
        "addr": ["WS13 6DP",
            "27 Lombard Street"],
        "email": ["fa@adcocks.com"]
    },
    "NER": {
        "LOC": ["Lichfield",
        "Staffordshire",
        "England"],
        "ORG": [
            "Adcocks Solicitors Limited",
            "Chancery House",
            "My Law Society",
            "Solicitors Regulation Authority"
        ],
        "PER": [
            "Hedley John Adcock",
            "representative"
        ]
    },
    "url": "http://solicitors.lawsociety.org.uk/person/5862/hedley-john-adcock"
}